UI/MapWnd.cpp

```diff
@@ -5592,7 +5592,12 @@ std::vector<int> MapWnd::FleetIDsOfFleetButtonsOverlapping(int fleet_id) const {
 
     const auto& it = m_fleet_buttons.find(fleet_id);
     if (it == m_fleet_buttons.end()) {
-        ErrorLogger() << "Couldn't find a FleetButton for fleet " << fleet_id;
+        auto vis_turn_map = GetUniverse().GetObjectVisibilityTurnMapByEmpire(fleet_id, HumanClientApp::GetApp()->EmpireID());
```

Comment:
comment this block? why / what is it checking?

Reply:
The change is just augmenting the preexisting logging that happens when the MapWnd has when it is told to select a fleet (such as if a fleet reference in a SitRep is clicked) but it can't find a corresponding FleetButton.  The newly added info is simply the information about when the fleet had last been seen, if at all. This was helpful to me as part of debugging the problem here.

So, a comment could be 
```
// log that a FleetButton could not be found for the requested fleet, and include when the fleet was last seen
```

Reply:
would be helpful